source/_components/climate.markdown

```diff
@@ -54,7 +87,7 @@ automation:
 
 ### {% linkable_title Service `climate.set_away_mode` %}
 
-Set away mode for climate device. The away mode changes the target temperature permanently to a temperature 
+Set away mode for climate device. The away mode changes the target temperature permanently to a temperature
```

Comment:
In my opinion away mode should change the temperature, but should also restore it when removed to previously set value.